### DISCUSSION TITLE
New Instrument Icon Designs For Review

### DISCUSSION THREAD

**Discussion by Riley-I** [jamulussoftware/jamulus#2437] [UI Related]
Hi all, 

Please visit link below to review recent icon drafts for review. Please comment feedback/critique. I am fixing sizing issues currently and will be re-uploading the files shortly. I am specifically concerned at how these will look at very small scale. Any recommendations greatly appreciated.

https://github.com/jamulussoftware/jamulus/pull/2435

![Jamulus-Icons](https://user-images.githubusercontent.com/95878117/155852740-6608ec1d-57d1-4ee0-90cb-5905310673e3.png)

**Comment by ann0see** [comment]
@jamulussoftware/designers I think this discussion is something for you 😀

Also cross-mentioning https://github.com/jamulussoftware/jamulus/issues/899

https://github.com/jamulussoftware/jamulus/issues/899 should be closed if a PR is merged.

**Comment by ann0see** [comment]
@eeconn @Alberstein8 since you were related to the issues with new instrument design, could you please comment here?

**Comment by jujudusud** [comment]
I like the idea of changing the icons that represent the instruments.
From an artistic point of view, I find them very successful. Anyway, I like it!
Regarding the integration in the windows, I think you should use the svg format instead of png if possible and think about the fact that the background of the windows can be different (white, black ... pink tomorrow?).
Good Job ;-)

**Reply by Riley-I** [reply]
Thanks for the feedback! I've uploaded svgs as well so that will always be an option in the future:)
https://github.com/jamulussoftware/jamulus/pull/2435

**Reply by pljones** [reply]
> pink

We already have white, yellow green and ... well ... it's meant to be red but it's very pale... which makes it pink, yes? 😀 

Existing style has enclosed parts of the icons solid colour, to help them stand out against the pale backgrounds.
